database/src/schema/account.rs

```diff
@@ -110,6 +110,21 @@ impl Account {
         .map(|_| ())
     }
 
+    // Delete an account
```

Comment:
```suggestion
    // Delete an account.
```
Grammar police arrived.

Reply:
Alright, don't mind me :smile:.
I was thinking that either all documentation should be full sentences or they shouldn't. As a german speaking person I feel uncomfortable around sentence fragments.

But you are the better judge of these kinda things, we aren't writing in german after all.

Reply:
This is a sentence fragment, thus no period. A full sentence would look like "This function deletes an account" or "delete() removes an account from the database". I feel like those are too verbose, so I vote no period :)

Reply:
I'm going to lean towards the more brief summaries for now. I feel like when you're reading the documentation, the subject is already implied as being the item you're documenting, so I don't like the idea of repeating or referring to it in the summary because it's redundant information. For that reason, I prefer a fragment as a "summary" of the item, usually followed by full sentences documenting it, if needed.

Let's agree to be a bit fluid early on with this. We haven't enabled the warning to document everything yet -- let's settle on a style when we do. In your next PR, try to document an item the way you normally would and we'll compare.

Reply:
Sounds good.